dist/focus-ring.js

```diff
@@ -265,7 +225,6 @@ document.addEventListener('DOMContentLoaded', function() {
     if (index(el).contains('focus-ring'))
       return;
     index(el).add('focus-ring');
-    el.setAttribute('data-focus-ring-added', '');
```

Comment:
Because I couldn't work out why it was necessary in the first place. Isn't it superfluous? Doesn't the presence of the `focus-ring` class give us enough?